test/runtests.jl

```diff
@@ -41,7 +41,9 @@ cd(dirname(@__FILE__)) do
                     test = shift!(tests)
                     local resp
                     try
-                        resp = remotecall_fetch(t -> runtests(t), p, test)
+                        # FIXME: `remote_fetch` doesn't work when
+                        #   a new module is define.
+                        resp = remotecall_fetch(t -> runtests(t, t != "examples"), p, test)
```

Comment:
One option is to split `examples.jl` and move the clustermanager and dictchannel tests to `parallel_exec.jl`. With an appropriate comment in `examples.jl` explaining the rationale for the move.